docs/enterprise-customer-facing-guide/setting-up-sso.md

```diff
@@ -0,0 +1,145 @@
+              Cam
+            </saml2:AttributeValue>
+        </saml2:Attribute>
+        <saml2:Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
+            <saml2:AttributeValue xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="xs:string">
+              Saul
+            </saml2:AttributeValue>
+        </saml2:Attribute>
+        <saml2:Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
+            <saml2:AttributeValue xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="xs:string">
+              cam@metabse.com
+            </saml2:AttributeValue>
+        </saml2:Attribute>
+    </saml2:AttributeStatement>
+</saml2:Assertion>
+```
+
+Most SAML identity providers we've used already include these
+assertions by default, but some (such as Okta) must be configured to
+include them. Here's an example of what your assertions configuration
+should look like in Okta. (You can find this page by going to `Admin > Applications > Metabase > General > SAML Settings [Edit]`).
+
+![Okta SAML Integration](images/saml-okta-setup.png)
+
+You can use other attribute names for these attributes if so desired;
+see the section below. The important thing is that first name (given
+name), last name (surname), and email address are included as
+attributes of the first assertion returned in the identity provider's
+SAML response.
+
+##### IMPORTANT NOTE!
+
+The email address *attribute* is used to log in an end user into a
+corresponding Metabase account (creating it if needed). Thus it is
+extremely critical that this email address MUST NOT be editable by end
+users themselves. Otherwise they could potentially access Metabase
+accounts other than their own by changing their email address.
+
+#### Settings for signing SSO requests (optional)
+These are additional settings you can fill in to sign SSO requests to
+ensure they don’t get tampered with.
+
+**SAML keystore path:** the absolute path to the keystore file to use for signing SAML requests.
+
+**SAML keystore password:** if it wasn't already self-evident, this is just the password for opening the keystore.
+
+**SAML keystore alias:** the alias for the key that Metabase should use for signing SAML requests.
+
+#### Settings for user attribute configuration (optional)
+These settings allow Metabase to automatically get each user's email address and first and last name.
+
+The settings that Metabase defaults to here might work for you out of the box, but you can override them if you know that your settings are different.
+
+Each of these input boxes needs a URI that points to the location of a SAML attribute.
+
+
+
+
+## JWT-based Authentication
```

Comment:
I think maybe the double `##` here isn't clear enough. It "feels" like it runs on from the SAML section vs being a different option